bitnami/milvus/Chart.yaml

```
@@ -48,4 +48,4 @@ maintainers:
 name: milvus
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/milvus
-version: 4.0.0
+version: 4.0.1
```

You are adding new parameters, please bump the chart to a new minor version

```suggestion
version: 4.1.0
```